### DISCUSSION TITLE
Conversion to ONNX for Javascript

### DISCUSSION THREAD

**Discussion by Rizialdi** [danielgatis/rembg#123] [Q&A]
Hi everyone,

I hope you are fine. 
I discovered this project yesterday, and I would to say congrats to you guys. It looks amazing.

I currently have a side-project (a chrome extension) running on the browser. Hence, I would to integrate one model, running on client-side for background removal. After some search on  Google I saw that it was possible to use ONNX format of a model for doing inference. 
However I don't know how to convert rembg to ONNX, nor if it's possible to do it. The alternative is either serving it through an api (hosting server or lambda function). 

Could you put some light on it ?
Is it possible to integrate rembg to a Javascript application ?

Best regards.

**Comment by danielgatis** [comment]
Now rembg use onnx models.
Look at:
https://github.com/danielgatis/rembg/blob/main/rembg/detect.py#L14
